backend/src/crud/crud_user.py

```diff
@@ -20,6 +20,7 @@ async def create(self, db: AsyncSession, *, obj_in: UserCreate) -> User:
             full_name=obj_in.full_name,
             created_at=datetime.now(),
             is_active=True,
+            is_admin=False

```

Comment:
this should be a value that gets passed in, not from obj_in, probably separately as another function param, check our crud creates to see how that works